<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>PHONE BOOK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 235, 155);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>(combine with PostgreSQL)</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>741</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 235, 155);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 235, 155);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 235, 155);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>750</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 4);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="explore_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 235, 155);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>EXPLORE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>REFRESH</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>id</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="surname">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>surname</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>phone</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="about">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Info about person</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_add_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="search_tag">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>600</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Search mark</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="search_info">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>600</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Search info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_search_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="delete_tag">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Delete mark</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="delete_info">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>720</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Delete info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_delete_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>720</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="change_tag">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>660</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Change mark</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="change_info">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>660</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Change info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_change_button">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="change_id_person">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>id</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
